examples/runtime-sdk/rofl-oracle-sgx/rofl.yaml

```diff
@@ -0,0 +1,30 @@
+name: rofl-oracle-sgx
+version: 0.1.0
+tee: sgx
+kind: raw
+resources:
+  memory: 512
+  cpus: 1
+  storage:
+    kind: disk-persistent
+    size: 512
```

Comment:
Persistent disk for SGX? That shouldn't work (it is probably ignored but it would be better if it failed).

Reply:
Yeah, this is just the manifest generated by `oasis rofl init --tee sgx --kind raw`.